joint-discovery-gatk4.hg38.wgs.inputs.json

```diff
@@ -1,92 +1,96 @@
-{
+{  
   "##_COMMENT1": "INPUT GVCFs & COHORT -- DATASET-SPECIFC, MUST BE ADAPTED",
-  "JointDiscovery_GATK4.cohort_vcf_name": "NA12878_hg38",
-  "JointDiscovery_GATK4.input_gvcfs_list": "gs://gatk-test-data/wgs_gvcf/PlatinumGenomes_hg38/PlatinumGenomes_hg38.gvcf_list",
-  "JointDiscovery_GATK4.input_gvcfs_indices_list": "gs://gatk-test-data/wgs_gvcf/PlatinumGenomes_hg38/PlatinumGenomes_hg38.gvcf_index_list",
-  "JointDiscovery_GATK4.sample_name_map" : "gs://gatk-test-data/wgs_gvcf/PlatinumGenomes_hg38/PlatinumGenomes_hg38.sample_map",
+  "JointGenotyping.callset_name": "String",
+  "JointGenotyping.sample_name_map": "File",
 
   "##_COMMENT2": "REFERENCE FILES",
-  "JointDiscovery_GATK4.ref_dict": "gs://broad-references/hg38/v0/Homo_sapiens_assembly38.dict",
-  "JointDiscovery_GATK4.ref_fasta": "gs://broad-references/hg38/v0/Homo_sapiens_assembly38.fasta",
-  "JointDiscovery_GATK4.ref_fasta_index": "gs://broad-references/hg38/v0/Homo_sapiens_assembly38.fasta.fai",
+  "JointGenotyping.ref_fasta": "gs://broad-references/hg38/v0/Homo_sapiens_assembly38.fasta",
+  "JointGenotyping.ref_fasta_index": "gs://broad-references/hg38/v0/Homo_sapiens_assembly38.fasta.fai",
+  "JointGenotyping.ref_dict": "gs://broad-references/hg38/v0/Homo_sapiens_assembly38.dict",
 
   "##_COMMENT3": "INTERVALS",
-  "JointDiscovery_GATK4.calling_intervals_list": "gs://gatk-test-data/intervals/wgs_calling_regions.hg38.intervals",
+  "JointGenotyping.eval_interval_list": "gs://broad-references/hg38/v0/wgs_evaluation_regions.hg38.interval_list",
+  "JointGenotyping.unpadded_intervals_file": "gs://gatk-test-data/intervals/hg38.even.handcurated.20k.intervals",
```

Comment:
this file should also be moved to our references bucket.  what do you think @eitanbanks